### ISSUE TITLE
Can't compile a project depending on Tlc5940Mux library

### ISSUE THREAD

**Issue by Kernald** [amperka/ino#56]
Hi,
Trying to compile a project depending on Tlc5940 library (from http://code.google.com/p/tlc5940arduino/) is working fine. However, using the Mux version, the dependencies are not detected, ino hangs:

```
Scanning dependencies of Tlc5940Mux
cat  > .build/uno/Tlc5940Mux/dependencies.d;
```

And nothing after.

**Comment by Kernald** [comment]
I'm pretty sure the issue comes from here: https://github.com/amperka/ino/blob/master/ino/make/Makefile.deps.jinja#L16

I created a simple Tlc5940Mux.cpp which is only including Tlc5940Mux.h in the lib folder. Now it works fine.

**Comment by nkrkv** [comment]
Fixed in 0.3.6
